app/views/layouts/_navbar.erb

```diff
@@ -14,6 +14,13 @@
         <li class='nav-item'>
           <a class='nav-link' href='/logout' data-method='post'>Logout</a>
         </li>
+      <% elsif @is_teacher %>
+        <li class='nav-item'>
+          <a class='nav-link' href='/'>Home</a>
```

Comment:
Do logged-in teachers really need a "Home" in the nav-bar? This functionality seems to be not well defined, as home allows users to create a new application, which is already email-specific, so log out seems to do the same thing.

Reply:
Additionally, if you do choose take out the Home, then you could probably clean up some of the logic in these stacked if statements to share more code.

Reply:
good point! we should ask michael what he wants to keep